static/css/main.css

```diff
@@ -89,18 +85,6 @@ a:hover .missing_image {
     border-bottom: 1px solid #e4e9fe;
 }
 
-.sorter_icon {
```

Comment:
I assume we were just duplicating this from elsewhere?

Reply:
Yes, it was exactly duplicated from Bootstrap's `.glyphicon` ruleset.

![Screenshot_2020-01-14_11-32-17](https://user-images.githubusercontent.com/202916/72336675-8d928800-36c1-11ea-9abe-18db35806368.png)